all-actors/src/main/java/org/sunbird/cert/actor/CertificateGeneratorActor.java

```diff
@@ -98,23 +98,16 @@ private void cleanup() {
 
     private String upload(String certFileName, String orgId, String batchId) {
         try {
-            //TODO  Un comment this to use cloud storage jar to upload file to azure as of now
-            // not using because of jar conflict issue
-
-            //HashMap<String,String> properties = new HashMap<>();
-            //properties.put(JsonKey.CONTAINER_NAME,System.getenv(JsonKey.CONTAINER_NAME));
-            //properties.put(JsonKey.CLOUD_STORAGE_TYPE,System.getenv(JsonKey.CLOUD_STORAGE_TYPE));
-            //properties.put(JsonKey.CLOUD_UPLOAD_RETRY_COUNT,System.getenv(JsonKey.CLOUD_UPLOAD_RETRY_COUNT));
-            //properties.put(JsonKey.AZURE_STORAGE_SECRET,System.getenv(JsonKey.AZURE_STORAGE_SECRET));
-            //properties.put(JsonKey.AZURE_STORAGE_KEY,System.getenv(JsonKey.AZURE_STORAGE_KEY));
-
-            //StorageParams storageParams = new StorageParams(properties);
-            //storageParams.init();
-            //return storageParams.upload(System.getenv(JsonKey.CONTAINER_NAME), "/", file, false);
-
             File file = FileUtils.getFile("conf/certificate/" + certFileName);
-            logger.info("CertificateGeneratorActor:upload:container name got from env is: ".concat(certVar.getCONTAINER_NAME()));
-            return AzureFileUtility.uploadFile(certVar.getCONTAINER_NAME() + "/" + orgId + "/" + batchId, file);
+            HashMap<String,String> properties = new HashMap<>();
+            properties.put(JsonKey.CONTAINER_NAME,System.getenv(JsonKey.CONTAINER_NAME));
```

Comment:
System.getenv(JsonKey.CONTAINER_NAME) and other variable can be re-used
so create constant and initialise it once only.

Reply:
done